docs/release-notes/remove-dry-run-det-deploy-aws.rst

```diff
@@ -0,0 +1,4 @@
+**Breaking Changes**
+
+-  CLI: The `--dry-run` option for `det deploy aws` has been removed. Previously, this option had no
+   effect, as AWS CloudFormation does not provide a utility to preview staged changes.
```

Comment:
**Breaking Changes**

-  CLI: Remove the ``--dry-run`` option for ``det deploy aws``. This option has no effect because AWS CloudFormation does not provide a utility to preview staged changes.